src/table/TableBodyWrapper.jsx

```diff
@@ -37,13 +37,13 @@ const TableBodyWrapper = ({
   selectionsAPI,
   layout,
   theme,
+  bodyHeight,
   focusedCellCoord,
   setShouldRefocus,
 }) => {
   const { rows, columns } = tableData;
   const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
   const bodyStyle = useMemo(() => getBodyStyle(layout, theme), [layout, theme.name()]);
-  const bodyHeight = rootElement.clientHeight - rootElement.getElementsByTagName('thead')[0]?.clientHeight - 52;
   const classes = useStyles({
     ...bodyStyle,
     height: bodyHeight,
```

Comment:
Actually, I think it would be good if you add px here. I got into some trouble when concatenating strings inside makeStyles

Reply:
like this

```suggestion
    height: `${bodyHeight}px`,
```